features/environment.py

```diff
@@ -645,26 +645,30 @@ def build_debs_from_dev_instance(context: Context, series: str) -> "List[str]":
             "ubuntu-behave-image-pre-build-%s-" % series + time_suffix
         )
 
-        cloud_manager = context.config.cloud_manager
+        package_cloud_manager = cloud.LXDContainer(
+            machine_type=context.config.machine_type
```

Comment:
Can we just hardcode "lxd.container" here instead of passing the machine_type we're testing (which may not be "lxd.container")?